docs/UserGuide.md

```diff
@@ -303,110 +374,121 @@ Format: `clear`
+# 7. Troubleshooting
 
 Facing issues with SweetRewards? Here are some common problems and how to resolve them.
 
-### Technical Issues
-1. Application doesn't start
-   * 🚫 Problem: Double-clicking the jar file doesn’t open SweetRewards. 
-   * ✅ Solution: Ensure Java 11 or above is installed on your machine. Open a command terminal, navigate to the folder containing the jar file, and run java -jar sweetrewards-v1.2.jar. If the problem persists, verify that the downloaded jar file is not corrupted and re-download if necessary.
+## Technical Issues
+1. Application won't start<br>
+🚫 **Problem**: Double-clicking the jar file doesn’t open SweetRewards.<br> 
+✅ **Solution**: Verify that Java 11 or higher is installed. Open a command prompt, navigate to the application's directory, and execute java -jar sweetrewards-v1.2.jar. If issues persist, re-download the jar file to rule out corruption.
+
+
+2. GUI opens off-screen<br>
+🚫 **Problem**:  SweetRewards starts off the visible screen area when using multiple displays.<br>
+✅ **Solution**:  Locate and delete preferences.json in the application's folder to reset window positioning, then restart the application.
+
+## Operational Issues
+1. Incorrect Points Allocation<br>
+🚫 **Problem**: Points were mistakenly added to the wrong member.<br>
+✅ **Solution**: Use the edit command to adjust the points for both involved members. Replace the newly updated (wrong) number of points with the initial (correct) number of points and add the points for the new member again. For example, edit 2 p/50 to replace the wrongly updated points and addpoints 3 p/50 to add points to the correct member.<br>
+
 
-2. GUI opens off-screen
+2. Incorrect Member Information Entered<br>
+🚫 **Problem**: Incorrect information (e.g., phone number or email) was entered for a member.<br>
+✅ **Solution**: Correct the mistake by using the `edit` command to adjust points for both the affected and intended members, ensuring accuracy in the loyalty program.<br>
 
-   * 🚫 Problem: When using multiple screens, SweetRewards opens off the visible screen. 
-   * ✅ Solution: Delete the preferences.json file found in the same folder as the jar file. Restart SweetRewards.
 
-### Operational Questions
-1. Adding Points to Wrong Member
-    * 🚫 Problem: Points were mistakenly added to the wrong member. 
-    * ✅ Solution: Use the edit command to adjust the points for both involved members. Replace the newly updated (wrong) number of points with the initial (correct) number of points and add the points for the new member again. For example, edit 2 p/50 to replace the wrongly updated points and addpoints 3 p/50 to add points to the correct member.
+3. Orders Not Reflecting in Member's History<br>
+🚫 **Problem**:  Newly added orders are not reflected in a member's history.<br>
+✅ **Solution**: Verify that `addorder` was used correctly, including the correct member name and order details. This ensures all transactions are accurately recorded.<br>
```

Comment:
Oh! Good catch here. Solution should be click off the person can click back. Ill edit this!